README.md

```diff
@@ -105,6 +105,26 @@ the next 100 are measured for inference throughput.
 The input parameters to the test are used to generate permutations
 of tests with varying configurations.
 
+### The model
+As Neural Network, we use Pytorch's implementation of Resnet50. The script `imagenet/model_saver.py`
+can be used to generate the model for CPU or GPU. 'By navigating to the imagenet folder, it can be invoked with
+
+```bash
+python model_saver.py
+```
+
+to generate the model for CPU, or with
+
+```bash
+python model_saver.py --device=GPU
+```
```

Comment:
I might not be remembering correctly but when the resnet file for GPU is generated, it should have an indication that its a resnet gpu file such as resnet-GPU.pt? I generated the resent file for GPU but its just named resnet50.pt so just checking

Reply:
I am unable to reproduce this behavior, the models I see are named `resnet50.GPU.pt` and `resnet50.CPU.pt`. Can you run it and paste the full output?

Reply:
Ignore this bc I made a dumb mistake! It generates with .GPU and .CPU, but I do have another Q! When I run `srun -n 1 -C P100 python model_saver.py --device=GPU` or `srun -n 1 -C P100 python model_saver.py --device=CPU`, it generates the respective files correctly. But if I run `srun -n 1 -C P100 python model_saver.py` without the device flag to generate a resent50.CPU.pt file, it generates a GPU file. Im not sure if this is an issue in the code or in the documentation.

Reply:
Correct, I had the false default value!